RM_vs_DFN.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Import the PyBaMM and essentials packages, as well as a set of parameters.
</pre><p>This comment could be put as text before the code cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/brosaplanella/formation/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Done it